flowmachine/tests/conftest.py

```diff
@@ -221,6 +230,28 @@ def get_length(flowmachine_connect):
     yield lambda query: len(pd.read_sql_query(query.get_query(), con=get_db().engine))
 
 
+@pytest.fixture()
+def test_events_table(flowmachine_connect):
+    """Creates a test event table and grants"""
+    # Creds hard-coded here for two reasons; 1) they're public anyway 2) This fixture will crash when run live
+    with connections(flowdb_user="flowdb", flowdb_password="flowflow"):
```

Comment:
Get them from the environment vars instead.

Reply:
Will change, but as this is going away it's moot - see comment for https://github.com/Flowminder/FlowKit/pull/4470/files/9910196c325f5accac4e91563ed8771947694007#r740146136